packages/extension/src/components/DaDndMessage.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="dnd-message-bar">
+    <p class="do-not-disturb">Do not disturb.</p>
+    <a href="#" class="turn-off" role="button" @click="$emit('dndOff')">Turn Off</a>
```

Comment:
This better be a `button` element because it is not a link but an actual button.

Reply:
Oh my bad 😅